docs/dev/analysis/features/table/table-cell.rst

```diff
@@ -29,6 +43,15 @@ MS API - Partial Summary
 * WordWrap
 
 
+WD_VERTICAL_ALIGNMENT Enumeration
+---------------------------------
+
+* wdAlignVerticalTop (0) Top vertical alignment.
+* wdAlignVerticalCenter (1) Center vertical alignment.
+* wdAlignVerticalJustify (2) Justified vertical alignment.
+* wdAlignVerticalBottom (3) Bottom vertical alignment.
+
+
```

Comment:
We should add an example using vertical alignment to the XML specimen below. We'll want a known-correct example handy when we get into the implementation :)